app/components/landingpage/Hero.js

```diff
@@ -0,0 +1,116 @@
+import Image from "next/image";
+import React from "react";
+import { register } from "swiper/element/bundle";
+
+register();
+
+function Hero() {
+  return (
+    <section className="w-full h-screen p-8 space-y-3 bg-black text-white text-center">
+      <div className="space-y-3 mt-4">
+        <p className="md:text-5xl text-2xl font-lexend ">
+          Get Your Confidence Again
+        </p>
+        <p className="md:text-2xl text-[10px]">
+          Discover the latest trends and showcase your unique style
+        </p>
+
+        <button className="p-3 bg-white text-black" type="button">
```

Comment:
Kindly import the reusable button from the `ui` directory and use that instead

Reply:
Done